code/modules/mob/living/carbon/human/life.dm

```diff
@@ -397,6 +405,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 		if(drunkenness >= 61)
 			if(prob(50))
 				blur_eyes(5)
+			if(has_trait(TRAIT_DRUNK_HEALING))
+				adjustBruteLoss(-0.5, 0)
+				adjustFireLoss(-0.25, 0)
```

Comment:
I'm worried these values might be a little high but being drunk has downsides and this is a 2-point trait anyways so it's probably no big deal.

Reply:
I'm cool with lowering this a little bit to err on the side of caution, although as you say this comes with heavy downsides and it already takes about 1 minute for drunkness to build up to that. At that stage there's heavy confusion, vomiting,  blurred eyes and you start having minor liver damage. 
It's really not a good idea to get to that level but I figured if someone is really desperate I may as well reward them for incapacitating themselves for several minutes.